packages/frontend/tests/integration/components/reports/new-subject-test.js

```diff
@@ -109,7 +138,21 @@ module('Integration | Component | reports/new-subject', function (hooks) {
 
   test('selecting and de-selecting a MeSH term as prepositional object', async function (assert) {
     this.server.create('mesh-descriptor');
-    await render(hbs`<Reports::NewSubject @close={{(noop)}} />`);
+    this.set('selectedPrepositionalObject', null);
+    this.set('setSelectedPrepositionalObject', (object) => {
+      this.set('selectedPrepositionalObject', object);
```

Comment:
i'd throw test assertions to check the input into all of these setters. to verify that the correct value is passed when the component's controls are under test.